igcollect/switch.py

```diff
@@ -115,6 +137,11 @@ def main():
     monitored_ports = get_monitored_ports(snmp, model)
     ports_stats(args.prefix, snmp, monitored_ports, model)
 
+    # We check DOM metrics only for switch models that have OIDs added to
+    # the script
+    if model in DOM_OIDS:
+        dom_stats(args.prefix, snmp, monitored_ports, model)
```

Comment:
this is a minor, but you could already pass `DOM_OIDS[model]` into the function to decouple it from this scope

Reply:
Good idea, I adjusted that.